classes/04class/exercises/c04-iac02/mfreitassm/main.tfvars

```diff
@@ -0,0 +1,13 @@
+aws_region = "ap-southeast-2"
+
+cidr_block_vpc = "10.0.0.0/16"
+
+private_subnet_cidr_blocks = ["10.0.0.0/24", "10.0.10.0/24"]
+private_subnet_names       = ["private-a", "private-b"]
+
+
+public_subnet_cidr_blocks = ["10.0.1.0/24", "10.0.11.0/24"]
+public_subnet_names       = ["public-a", "public-b"]
+
+
+availability_zones = ["ap-southeast-2a", "ap-southeast-2b"]
```

Comment:
Using data sources helps you to avoid these inputs that can be inferred.

Have a look at this one for availability zones:
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/availability_zones